demos/prefsFramework/index.html

```diff
@@ -93,14 +95,22 @@
         <div class="flc-overviewPanel fl-overviewPanel-container"></div>
         <!-- BEGIN markup for Preference Editor -->
         <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!-- This div is for the sliding panel that shows and hides the Preference Editor controls in the mobile view-->
+            <div class="fl-panelBar fl-panelBar-above">
```

Comment:
@cindyli one is above the sliding panel and one is below. The one above is used for mobile and the one below for desktop. It is possible to have a single one and change the placement with styling; however, doing so will break the tab order for one of the implementations. I had spoken to Lisa about that, and she indicated that it would be a WCAG failure. In the future we plane to only have the panel bar above, in all cases. However, there needs to be some design consideration worked out for how this will affect the interactions on the desktop view first.  Meaning that the change won't happen in the near future.